app/Helpers/helpers.php

```diff
@@ -221,7 +221,9 @@ function milestone_completion_notification($task_id){
 
 function project_paused_notification($project_id){
     $project = Project::where('id', $project_id)->first();
-    $estimate = Estimate::where('project_id', $project_id)->first();
+    $estimate = Estimate::where('project_id', $project_id)
+                        ->where('status', 'project_started')
```

Comment:
When the estimate is accepted by the customer then I don't think the estimate status is changed to project_started

Reply:
Line 341: Same changes to be made as in this one

Reply:
resolved